test/test_util/test_util.dart

```diff
@@ -14,7 +14,4 @@
 
 library test_util;
 
-export 'custom_matchers.dart';
-export 'dom_util.dart';
-export 'react_util.dart';
-export 'wrapper_component.dart';
+export 'package:over_react_test/over_react_test.dart';
```

Comment:
I just exported these instead of updating the imports in every file, in order to get this through more quickly and with less risk of merge conflicts.